### DISCUSSION TITLE
Changing the faker locale when using a factory

### DISCUSSION THREAD

**Discussion by jansgescheit** [laravel/framework#41200] [Ideas]
I just stumbled across it while seeding. It would be nice if it could match the locale of the faker instance to a factory method.
Of course I can create a new faker in the definition. Maybe I could override it in a state too.

But something like this would be cool:

`Post::factory()->faker('de')->create(4)`

**Comment by gguney** [comment]
You can use states as you said. Maybe you can add a argument to your custom state and customize it in the factory?
